frontend-next-migration/src/entities/Hero/ui/BarChart/BarChart.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from '@storybook/react';
+import React from 'react';
+import { BarChart, BarChartProps } from './BarChart';
+
+const meta: Meta<typeof BarChart> = {
+    title: 'Entities/Hero/BarChart',
+    component: BarChart,
+    argTypes: {
+        width: {
+            description: 'width of bar chart',
+        },
+        height: {
+            description: 'height of bar chart',
+        },
+        stats: {
+            description: 'stats',
+        },
+    },
+    args: {
+        width: 400,
```

Comment:
I'm a little confused that this needs to be hardcoded(width, height), it wouldn't be convenient to make it adaptive, would it?